common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -178,6 +178,17 @@
                 @NewNavigation.BrandExtensions.getEditionalisedNavLinks(edition).map { item =>
                     @brandExtensions(item)
                 }
+
+                @if(HeaderFeedback.isSwitchedOn) {
+                    <li class="hide-until-desktop feedback-form">
+                        @fragments.inlineSvg("information-circle", "icon", List("feedback-form__icon"))
+                        This navigation is new. We'd appreciate your feedback.
```

Comment:
Not much visual difference in this particular font, but my nature got better of me: we should use proper apostrophe `U+2019` (or `&rsquo;`), instead of an impostor. Thank you for your cooperation.

Reply:
You rule! 🌴